CRTX-217342 | Mapping for iManage Threat Manager (#44595)
demisto/content: Packs/iManageThreatManager/ReleaseNotes/1_0_3.md

#### Modeling Rules

##### New: iManage Threat Modeling Rule

> - New: Added a new modeling rule- iManage Threat Modeling Rule to enable normalization of iManage Threat Manager data ingested via API into Cortex XSIAM for investigation and analysis.

Reviewer: ```suggestion
Added the **iManage Threat Modeling Rule** to enable normalization of iManage Threat Manager data ingested via API into Cortex XSIAM for investigation and analysis.
```
